post.js

```diff
@@ -93,7 +107,7 @@ class DistributedPost extends HTMLElement {
 
     // Published time element
     const publishedTime = document.createElement('a')
-    const postUrl = jsonLdData.id || jsonLdData.object.id
+    const postUrl = fetchLink({ url: jsonLdData.id || jsonLdData.object.id, id: jsonLdData.id || jsonLdData.object.id })
```

Comment:
```suggestion
    const postUrl = fetchLink(jsonLdData.id)
```

jsonLdData at this point should only be a note. Show an error if an Activity gets passed in the url.